lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -267,6 +257,35 @@ def cmd_banner(*args)
 
   end
 
+  def cmd_tip_help
+    print_line "usage: tip [options]"
+    print_line
+    print_line "Print a useful tip on how use metasploit"
+    print @@tip_opts.usage
+  end
+
+  #
+  # Display a useful productivity tip to the user.
+  #
+  def cmd_tip(*args)
+    if args.include?("-h")
+      cmd_tip_help
+    elsif args.include?("-l")
+      tbl = Table.new(
+        Table::Style::Default,
+        'Columns' => %w(Id Tip)
```

Comment:
We prefer [%w[]](https://github.com/rubocop-hq/ruby-style-guide#percent-literal-braces) for arrays. It's a little clearer it's not a quoted string, etc.

Reply:
Good catch! I can't wait to have rubocop enabled haha